core/utils/type-utils/src/main/java/datawave/data/type/BaseType.java

```diff
@@ -9,16 +9,25 @@
 
 public class BaseType<T extends Comparable<T> & Serializable> implements Serializable, Type<T>, ObjectSizeOf {
     
-    private static final long serialVersionUID = 5354270429891763693L;
+    private static final long serialVersionUID = 1261391800060720465L;
     private static final long STATIC_SIZE = PrecomputedSizes.STRING_STATIC_REF + Sizer.REFERENCE + Sizer.REFERENCE;
     
     protected T delegate;
     protected String normalizedValue;
     protected final Normalizer<T> normalizer;
     
-    public BaseType(String delegateString, Normalizer<T> normalizer) {
+    /**
+     * Default constructor. Generates the normalized and non-normalized variants of the provided value
+     * 
+     * @param value
+     *            the value
+     * @param normalizer
+     *            the normalizer
+     */
+    public BaseType(String value, Normalizer<T> normalizer) {
         this.normalizer = normalizer;
-        setDelegate(normalizer.denormalize(delegateString));
+        this.delegate = normalizer.denormalize(value);
+        this.normalizedValue = normalizer.normalize(value);
```

Comment:
So previously we set the normalized value to the normalization of the denormalized value.  This change is slightly different and I wonder if there may be an unintended consequence.

Reply:
Good catch, next commit functionally restores the original code path. Rebuilt against main datawave and all tests pass.